internal/retryhttp/client.go

```diff
@@ -67,6 +68,12 @@ func (c *Client) Do(req *retryablehttp.Request) (*http.Response, error) {
 	return normalizeResponse(c.Client.Do(req))
 }
 
+func (c *Client) StandardClient() *http.Client {
```

Comment:
What invokes this? And wouldn't this cause some sort of recursion in the retrying mechanic?

Reply:
It's invoked [here](https://github.com/ActiveState/cli/blob/master/pkg/platform/api/graphql/graphql.go#L50) and various other places. Before this was using the `StandardClient` function from the retry library and not ours.

I don't think there should be any recursion issues as this is still using our client in the round tripper. I noticed in the retry library they use `sync.once` when initializing the round tripper and the retry client. We could do something similar here just to be sure.